docs/quick-start/download-and-install.txt

```diff
@@ -28,6 +28,19 @@ Download and Install the Dependencies
 Complete the following steps to install and add the {+odm-short+} dependencies
 to a Laravel web application.
 
+.. tip::
+
+   Alternatively, you can use Laravel Herd to install MongoDB and configure a
+   Laravel MongoDB development environment. For more information about using
+   Laravel Herd with MongoDB, see the following resources:
```

Comment:
This admonition box leads with "Alternatively..." before introducing any of the dependencies in the main body of the page. Would consider either moving this tip box lower or updating the text to the following.

Let me know if you'd like to talk more about this feedback!
```suggestion
.. tip::

   As an alternative to the steps detailed on this page, you can use Laravel Herd to install MongoDB and configure a
   Laravel MongoDB development environment. For more information about using
   Laravel Herd with MongoDB, see the following resources:
```

Reply:
I kept the admonition where it is but reworded - lmk what you think!